.github/workflows/autofixes.yml

```diff
@@ -17,32 +20,19 @@ jobs:
         working-directory: e2e-tests
       - run: npm run autofix
       - name: Automated Linting
+        id: automated-linting
         uses: EndBug/add-and-commit@v9
         with:
           message: "Automated linting update and features.json sync"
           add: "*"
           push: true
           pull: --rebase --autostash
           default_author: github_actions
+      - name: Fail on linting push
+        run: |
+          if [[ ${{ steps.automated-linting.outputs.pushed }} == true ]]; then exit 1; else exit 0; fi
+
+          
+
+
```

Comment:
nit: looks like there are some extra newlines here

Reply:
oops how'd you guys get there

Reply:
you should be able to run prettier on the `.github` directory and it'll take care of it. although you'll need to temporarily remove the directory from the `.prettierignore`

Reply:
I think we only had it in the prettierignore before because we didn't have the repo_scoped_bot_token set up, also down to remove it!